.github/workflows/sync-from-upstream.yaml

```
@@ -0,0 +1,36 @@
+name: Sync from upstream
+
+on:
+  schedule:
+    - cron: '0 7 */7 * *' # At 07:00 on every 7th day-of-month
```

Have you tried the action by triggering it from the actions tab?

```suggestion
on:
  schedule:
    - cron: '0 7 */7 * *' # At 07:00 on every 7th day-of-month

  # Allows to run this workflow manually from the Actions tab
  workflow_dispatch:
```

---

Can I run manually from a PR? Or do I need to merge first? :P

---

mh looks like it needs a merge first :(